public/app/plugins/datasource/influxdb/datasource.ts

```diff
@@ -802,3 +814,15 @@ function timeSeriesToDataFrame(timeSeries: TimeSeries): DataFrame {
     length: values.length,
   };
 }
+
+export function influxRegularEscape(value: any) {
```

Comment:
any idea to not use `any` here @joshhunt ?

Reply:
grafana data exports `TypedVariableModel` which can be narrowed down with either `if ("multi" in variable)` checks, or `if (variable.type === "query")`

Reply:
I've double checked the types that `templateSrc.replace` needs and I ended up doing this: https://github.com/grafana/grafana/pull/75527/commits/eec6b90f5f5e6bf9caeb904f465f792603dff3eb

Reply:
should this be imported from scenes? feels weird imho

Reply:
I just used the same logic from prometheus https://github.com/grafana/grafana/blob/main/public/app/plugins/datasource/prometheus/datasource.tsx#L335-L352
Where is this in scenes?

Reply:
you wrote `import { CustomFormatterVariable } from '@grafana/scenes';` in your PR. It feels weird to me that this type is being imported from @grafana/scenes, but i see template_srv does so 🤷🏻